src/cfg/to_bril.rs

```diff
@@ -0,0 +1,129 @@
+use bril_rs::{Code, EffectOps, Function, Instruction, Program};
+
+use super::{Annotation, BasicBlock, SimpleBranch, SimpleCfgFunction, SimpleCfgProgram};
+use petgraph::{
+    stable_graph::NodeIndex,
+    visit::{DfsPostOrder, Walker},
+};
+
+impl SimpleCfgProgram {
+    /// Converts the Cfg into a bril program.
+    pub fn to_bril(&self) -> Program {
+        let mut bril = Program {
+            functions: vec![],
+            imports: vec![],
+        };
+        for func in &self.functions {
+            bril.functions.push(func.to_bril());
+        }
+        bril
+    }
+}
+
+impl SimpleCfgFunction {
+    /// Converts the cfg function into a bril program.
+    pub fn to_bril(&self) -> Function {
+        // Make an empty function
+        let mut func = Function {
+            name: self.name.clone(),
+            args: self.args.clone(),
+            instrs: vec![],
+            pos: None,
+            return_type: self.return_ty.clone(),
+        };
+
+        // start with the entry block
+        self.node_to_bril(self.entry, &mut func);
+
+        // The order of this traversal does not matter, just need to loop over the blocks
+        DfsPostOrder::new(&self.graph, self.entry)
+            .iter(&self.graph)
+            .for_each(|node| {
+                // don't do the exit or entry
+                if node != self.entry && node != self.exit {
+                    // Add a label for the block
+                    func.instrs.push(Code::Label {
+                        label: format!("{}", self.graph[node].name),
```

Comment:
nit: I think it'd be easier to read (and refactor) if we had a separate `label_name`  function that converted an identifier to a label. We use this convention of `format!("{}"` a few different places in this file, and it could be difficult to clean up in the future if we needed to change the formatting for some reason.